new-log-viewer/src/contexts/StateContextProvider.tsx

```diff
@@ -82,7 +82,7 @@ const StateContextProvider = ({children}: StateContextProviderProps) => {
             case WORKER_RESP_CODE.PAGE_DATA:
                 setLogData(args.logs);
                 setLogLines(args.lines);
-                setLogEventNum(args.startLogEventNum);
+                setLogEventNum(args.lines.get(1) as number);
```

Comment:
* Sorry for the confusions. `logEventNum` was intended to replace `logEventIdx` to be consistent with the new naming convention, since the index starts from 1 rather than 0.
* Sure, I agree `beginLineNumToLogEventNum` is a better name.

Reply:
Hm, so if `logEventNum` is meant to be the log event requested by the user, is `setLogEventNum(args.lines.get(1) as number);` just meant to set the log event number to the first log event in the page? Why would we want to do that?

Reply:
It was a placeholder before Monaco can be integrated. To show the last event when the log file gets loaded, it should be this instead:
```
const lineNumbers = Array.from(args.beginLineNumToLogEventNum.values());
const lastLogEventNum = lineNumbers[args.beginLineNumToLogEventNum.size - 1];
if ("undefined" !== typeof lastLogEventNum) {
    setLogEventNum(lineNumbers[args.beginLineNumToLogEventNum.size - 1] as number);
} else {
    console.error("Last logEventNum expectedly undefined");
}
```

Reply:
Gotcha. But if the user requests a certain logEventNum, we should seek and show that one specifically rather than the last log event on the page, right?